2022-02-02 17:21:22,369 INFO    MainThread:3029 [wandb_setup.py:_flush():71] setting env: {}
2022-02-02 17:21:22,369 INFO    MainThread:3029 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-02 17:21:22,369 INFO    MainThread:3029 [wandb_init.py:_log_setup():371] Logging user logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220202_172122-1vewg664/logs/debug.log
2022-02-02 17:21:22,369 INFO    MainThread:3029 [wandb_init.py:_log_setup():372] Logging internal logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220202_172122-1vewg664/logs/debug-internal.log
2022-02-02 17:21:22,370 INFO    MainThread:3029 [wandb_init.py:init():404] calling init triggers
2022-02-02 17:21:22,370 INFO    MainThread:3029 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-02 17:21:22,370 INFO    MainThread:3029 [wandb_init.py:init():460] starting backend
2022-02-02 17:21:22,370 INFO    MainThread:3029 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-02 17:21:22,393 INFO    MainThread:3029 [backend.py:ensure_launched():216] starting backend process...
2022-02-02 17:21:22,406 INFO    MainThread:3029 [backend.py:ensure_launched():222] started backend process with pid: 3066
2022-02-02 17:21:22,406 INFO    MainThread:3029 [wandb_init.py:init():469] backend started and connected
2022-02-02 17:21:22,408 INFO    MainThread:3029 [wandb_init.py:init():533] updated telemetry
2022-02-02 17:21:22,443 INFO    MainThread:3029 [wandb_init.py:init():563] communicating current version
2022-02-02 17:21:22,956 INFO    MainThread:3029 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-02 17:21:22,957 INFO    MainThread:3029 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-02 17:21:23,226 INFO    MainThread:3029 [wandb_init.py:init():606] starting run threads in backend
2022-02-02 17:21:24,417 INFO    MainThread:3029 [wandb_run.py:_console_start():1810] atexit reg
2022-02-02 17:21:24,417 INFO    MainThread:3029 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-02 17:21:24,417 INFO    MainThread:3029 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-02 17:21:24,419 INFO    MainThread:3029 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-02 17:21:24,419 INFO    MainThread:3029 [wandb_init.py:init():633] run started, returning control to user process
2022-02-02 17:21:24,679 INFO    MainThread:3029 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-02 17:21:24,679 INFO    MainThread:3029 [wandb_run.py:_restore():1752] restore
2022-02-02 17:21:26,943 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 786
}

2022-02-02 17:21:27,261 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 786
}

2022-02-02 17:21:28,248 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 1107
}

2022-02-02 17:21:28,353 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 13160
}

2022-02-02 17:21:28,458 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 13160
}

2022-02-02 17:21:28,561 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 13160
}

2022-02-02 17:21:28,666 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:21:28,770 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:21:28,873 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:21:28,976 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:21:29,079 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:21:29,183 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:21:29,415 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:21:29,936 INFO    MainThread:3029 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}
local_info {
}

2022-02-02 17:21:31,000 INFO    MainThread:3029 [wandb_run.py:_append_files():2180] logging synced files
